### ISSUE TITLE
Generator does not work

### ISSUE THREAD

**Issue by artofhuman** [artofhuman/activeadmin_settings_cached#21]

**Comment by bzvyagintsev** [comment]
How can I use this gem now?

**Comment by artofhuman** [comment]
@bzvyagintsev U can just add admin settings file manually.
example:

```
# admin/settings.rb
ActiveAdmin.register_page 'Settings’ do
  menu label: 'Settings', priority: 99

  active_admin_settings_page(
    display: {
      'metrika' => :text
    }
  )
end
```

**Comment by okliv** [comment]
+1

**Comment by neilabdev** [comment]
+1  as doing it manually I got:

Uncaught exception: undefined method `active_admin_settings_page' for #ActiveAdmin::PageDSL:0x007fd604f0d4a8

**Comment by merqlove** [comment]
Hmm,  i don't have such bug.
Maybe you has old AA? Try to update it from its repo master.

**Comment by merqlove** [comment]
Rails 5
```ruby
bundle exec appraisal rails5.0 spec/rails/rails-5.0.1/bin/rails g settings:install
>> BUNDLE_GEMFILE=gemfiles/rails5.0.gemfile bundle exec spec/rails/rails-5.0.1/bin/rails g settings:install
   identical  app/models/setting.rb
    conflict  config/app.yml
Overwrite config/app.yml? (enter "h" for help) [Ynaqdh] n
        skip  config/app.yml
   identical  db/migrate/20161229111205_create_settings.rb
```

Rails 4
```ruby
bundle exec appraisal rails5.0 spec/rails/rails-4.2.7.1/bin/rails g settings:install
>> BUNDLE_GEMFILE=gemfiles/rails5.0.gemfile bundle exec spec/rails/rails-4.2.7.1/bin/rails g settings:install
   identical  app/models/setting.rb
    conflict  config/app.yml
Overwrite config/app.yml? (enter "h" for help) [Ynaqdh] n
        skip  config/app.yml
   identical  db/migrate/20161229102058_create_settings.rb
```

**Comment by merqlove** [comment]
@neilabdev @artofhuman @okliv @bzvyagintsev Hi, Merry Christmas:) 
Anyone still have this issue? 
Seems that current question is resolved.

**Comment by artofhuman** [comment]
Fixed via #32
